frontend_lib/src/component/CardPopup/CardPopupCreateContent.styl

```diff
@@ -51,9 +51,15 @@
         &:disabled
           cursor not-allowed
 
+.createcontent
+  &__contentname
+    margin-bottom 1.5rem
```

Comment:
This particular case was a code that there was a bootstrap 'mb-4' class that I preferred to put in the stylus file instead of leaving in the jsx, but I kept the exact same value. However, I am also not against using more rem in Tracim, since it is a slightly more responsive and accessible unit.

Reply:
ok
While I agree on the fact that rem is more responsive, it is also more complicated to use and can lead to edge cases when not used properly.
It is a more powerful tool but it require some consistency, so use with caution